### PR TITLE
replay: be more clear about the purpose of it

### PR DESCRIPTION
**Pull request by Stifael**

### CODE REVIEW

File: ekf2_log_replay.md [hunk 1]
```diff
@@ -1,30 +1,43 @@
 # EKF2 Log Replay
 
-This page shows you how you can tune the parameters of the EKF2 estimator by using the replay feature on a real flight log. It is based on the `sdlog2` logger.
+This page shows you how you can tune the parameters of the EKF2 estimator by using the replay feature on a real flight log.
 
 ## Introduction
+A developer has the possibility to do replay on logged data for estimation analysis. The remainder of this page will explain which parameters have to be set in order to benefit from this feature and how to correctly deploy it.
```
Comment (simonegu) [inline ekf2_log_replay.md]: `on particularly logged data` not each logged data is good to be replayed

**Review by bkueng — APPROVED** [review]

File: ekf2_log_replay.md [hunk 2]
```diff
@@ -1,30 +1,43 @@
...
+<path to Firmware>/build_posix_sitl_replay/src/firmware/posix/rootfs/replay_replayed.px4log
+```
+This replayed file can then be used to analyze the estimator performance. 
 
-## Changing tuning parameters for a replay
 
-You can set the estimator parameter values for the replay in the file **replay\_params.txt** located in the same directory as your replayed log file, e.g. **build\_posix\_sitl\_replay/src/firmware/posix/rootfs/replay\_params.txt**. When running the replay for the first time \(e.g. after a **make clean**\) this file will be auto generated and filled with the default EKF2 parameter values taken from the flight log. After that you can change any EKF2 parameter value by changing the corresponding number in the text file. Setting the noise value for the gyro bias would require the following line.
+### Changing tuning parameters for a replay
+
+You can set the estimator parameter values for the replay in the file **replay\_params.txt** located in the same directory as your replayed log file, e.g. **build\_posix\_sitl\_replay/src/firmware/posix/rootfs/replay\_params.txt**.
```
Comment (simonegu) [inline ekf2_log_replay.md]: Can you explain were the file go. build_positl_replay ...

File: ekf2_log_replay.md [hunk 3]
```diff
@@ -1,30 +1,43 @@
...
+When you run the replay the first time, the replay_replayed.px4log file will be generated and filled with the default EKF2 parameter values used during the actual flight.
```
Comment (simonegu) [inline ekf2_log_replay.md]: The parameter are saved in the `replay_params.txt` right and the computed log is `replay_replayed.px4log`

File: ekf2_log_replay.md [hunk 4]
```diff
@@ -1,30 +1,43 @@
...
+### Changing tuning parameters for a replay
+
+You can set the estimator parameter values for the replay in the file **replay\_params.txt** located in the same directory as your replayed log file, e.g. **build\_posix\_sitl\_replay/src/firmware/posix/rootfs/replay\_params.txt**.
+When you run the replay the first time, the replay_replayed.px4log file will be generated and filled with the default EKF2 parameter values used during the actual flight. 
+After that you can change any EKF2 parameter value by changing the corresponding number in the text file. Setting the noise value for the gyro bias would require the following line.
```
Comment (simonegu) [inline ekf2_log_replay.md]: For example

File: ekf2_log_replay.md [hunk 5]
```diff
@@ -1,30 +1,43 @@
...
 
 ```
 EKF2_GB_NOISE 0.001
 ```
+Once EKF2 parameter is changed, the replay can be run again with the new value.
```
Comment (simonegu) [inline ekf2_log_replay.md]: Add run command just to be clear.

**Pull request merged**